pkg/course/course.go

```diff
@@ -374,7 +436,22 @@ func (f *FileV2) populateEmptyChartNames() {
 	}
 }
 
-func (f FileV2) validateJsonSchema(schemaData []byte) error {
+// populateNamespaceManagement populates each release with the default namespace management settings if they are not set
+func (f *FileV2) populateNamespaceManagement() {
+	var emptyNamespaceMgmt NamespaceConfig
+	if f.NamespaceMgmt.Default == nil {
+		f.NamespaceMgmt.Default = &emptyNamespaceMgmt
+	}
+	for releaseIndex, release := range f.Releases {
+		if release.NamespaceMgmt == nil {
+			klog.V(5).Infof("using default namespace management for release: %s", release.Name)
+			release.NamespaceMgmt = f.NamespaceMgmt.Default
+			f.Releases[releaseIndex] = release
+		}
+	}
+}
```

Comment:
Check about merging vs. overwriting namespace_management and which takes precedence in the docs.